### DISCUSSION TITLE
SwissQRBill Browser Installation

### DISCUSSION THREAD

**Discussion by PCWIN2007** [schoero/SwissQRBill#389] [General]
Good day,

Do you also offer remote browser installation for SwissQRBill?

Thank you in advance.
Best regards pcwin

**Comment by schoero** [comment]
I don't understand what you mean by "remote browser installation".
Can you clarify?

**Reply by PCWIN2007** [reply]
Hello Mr. Schönbächler,
Thanks for your Answer.
Sorry for wording. I meant Browser installation via (remote access, Teamviewer).
Or I can provide you with the access data (FTP etc.).
I am willing to meet your commitment if possible, you can contact me directly by mail or phone at (pcwin.ch).

Thank you in advance.
Best regards Kallaba

**Reply by schoero** [reply]
I am not sure if you have not fully understood what this library is about.
This library offers a way for developers to create QR-Bills programmatically. It does not provide a graphical user interface or something similar that you can install in your browser.

So you could use this library to create QR bills for your company. That means you have to develop an interface to your accounting software and use this library to render the actual QR-Bill.

The development of such an interface is not something that I offer.
